<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>chifoumivue</class>
 <widget class="QMainWindow" name="chifoumivue">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>568</width>
    <height>563</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>568</width>
    <height>563</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>568</width>
    <height>563</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="Fight">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>140</y>
      <width>65</width>
      <height>65</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>65</width>
      <height>65</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>65</width>
      <height>65</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="ressourcesChifoumi.qrc">
      <normaloff>:/chifoumi/images/versus.gif</normaloff>:/chifoumi/images/versus.gif</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>65</width>
      <height>65</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="SelectJoueur">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>120</y>
      <width>111</width>
      <height>110</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>111</width>
      <height>110</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>111</width>
      <height>110</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="ressourcesChifoumi.qrc">
      <normaloff>:/chifoumi/images/rien.gif</normaloff>:/chifoumi/images/rien.gif</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>111</width>
      <height>110</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="SelectMachine">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>120</y>
      <width>111</width>
      <height>110</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>111</width>
      <height>110</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>111</width>
      <height>110</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="ressourcesChifoumi.qrc">
      <normaloff>:/chifoumi/images/rien.gif</normaloff>:/chifoumi/images/rien.gif</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>111</width>
      <height>110</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="BoutonPartie">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>480</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Nouvelle Partie</string>
    </property>
   </widget>
   <widget class="QLabel" name="ScoreJoueur">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>80</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="ScoreMachine">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>80</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="Joueur">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>40</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Vous</string>
    </property>
   </widget>
   <widget class="QLabel" name="Machine">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>40</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Machine</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>300</y>
      <width>489</width>
      <height>169</height>
     </rect>
    </property>
    <property name="title">
     <string>Choisissez une figure :</string>
    </property>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>491</width>
       <height>171</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="Ciseaux">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>111</width>
          <height>110</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>111</width>
          <height>110</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="ressourcesChifoumi.qrc">
          <normaloff>:/chifoumi/images/ciseau.gif</normaloff>:/chifoumi/images/ciseau.gif</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>111</width>
          <height>110</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Feuille">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>111</width>
          <height>110</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>111</width>
          <height>110</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="ressourcesChifoumi.qrc">
          <normaloff>:/chifoumi/images/papier.gif</normaloff>:/chifoumi/images/papier.gif</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>111</width>
          <height>110</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Pierre">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>111</width>
          <height>110</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>111</width>
          <height>110</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="ressourcesChifoumi.qrc">
          <normaloff>:/chifoumi/images/pierre.gif</normaloff>:/chifoumi/images/pierre.gif</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>111</width>
          <height>110</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>568</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>Fichier</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuaide">
    <property name="title">
     <string>Aide</string>
    </property>
   </widget>
   <addaction name="menuFichier"/>
   <addaction name="menuaide"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="ressourcesChifoumi.qrc"/>
 </resources>
 <connections/>
</ui>
